cvat-core/src/quality-settings.ts

```diff
@@ -19,12 +20,34 @@ export enum PointSizeBase {
     GROUP_BBOX_SIZE = 'group_bbox_size',
 }
 
+const settingsMapping: Record<string, string> = {
+    iouThreshold: 'iou_threshold',
+    oksSigma: 'oks_sigma',
+    pointSizeBase: 'point_size_base',
+    lineThickness: 'line_thickness',
+    lowOverlapThreshold: 'low_overlap_threshold',
+    orientedLines: 'compare_line_orientation',
+    lineOrientationThreshold: 'line_orientation_threshold',
+    compareGroups: 'compare_groups',
+    groupMatchThreshold: 'group_match_threshold',
+    checkCoveredAnnotations: 'check_covered_annotations',
+    objectVisibilityThreshold: 'object_visibility_threshold',
+    panopticComparison: 'panoptic_comparison',
+    compareAttributes: 'compare_attributes',
+    targetMetric: 'target_metric',
+    targetMetricThreshold: 'target_metric_threshold',
+    maxValidationsPerJob: 'max_validations_per_job',
+    emptyIsAnnotated: 'empty_is_annotated',
+    jobFilter: 'job_filter',
+    inherit: 'inherit',
+};
```

Comment:
Feels like it can be utility function to convert snake_case to camelCase
We already have `fieldsToSnakeCase` and Camelized<Type>, so, we may add reverse function and type.

Reply:
Removed this mapping, as new save method was implemented